### PR TITLE
[plugin.video.nfl.gamepass] 2018.08.15

### PR DESCRIPTION
**Pull request by aqw**
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

This updates the addon to work with the changes to the Game Pass service for the 2018 season.
Also, the [XML problem](https://github.com/xbmc/repo-plugins/pull/1380#issuecomment-347283743) caught by your automated checks should be fixed now.

Let me know if you have any concerns or questions.

---Alex

### CODE REVIEW

File: plugin.video.nfl.gamepass/addon.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
      marooned on an old version. -->
 <addon id="plugin.video.nfl.gamepass"
        name="NFL Game Pass"
-       version="0.11.1"
+       version="2018.08.15-release.1"
```
Comment (enen92) [inline plugin.video.nfl.gamepass/addon.xml]: I am not sure this is valid. @rechi can u help?

Comment (aqw) [reply plugin.video.nfl.gamepass/addon.xml]: @enen92 Ahh oops. I should have checked first. Let me know if it needs to be changed.

Comment (MartijnKaijser) [reply plugin.video.nfl.gamepass/addon.xml]: it might simply just work however it's not according to our x.y.z standard we try to keep

Comment (aqw) [reply plugin.video.nfl.gamepass/addon.xml]: @MartijnKaijser Thanks for the clarification. I've pushed a version with this fixed.
@enen92 I've squashed the fix and pushed it. It should be good now.

**Comment by aqw** [comment]
@enen92 Thanks for merging. :-)

**Pull request merged**